control-myplaces/src/main/java/fi/nls/oskari/control/myplaces/handler/MyPlacesLayersHandler.java

```diff
@@ -205,7 +174,7 @@ private MyPlaceCategory createDefaultCategory() {
         category.getWFSLayerOptions().setDefaultFeatureStyle(WFSLayerOptions.getDefaultOskariStyle());
         return category;
     }
-
+    @Deprecated
```

Comment:
Since it's a private method it should be safe to just remove it instead of deprecating it